src/pandas_profiling/model/pandas/describe_categorical_pandas.py

```diff
@@ -53,7 +53,14 @@ def counter_to_series(counter: Counter) -> pd.Series:
 
 
 def unicode_summary_vc(vc: pd.Series) -> dict:
-    from tangled_up_in_unicode import block, block_abbr, category, category_long, script
+    try:
+        from tangled_up_in_unicode import block, block_abbr, category, category_long, script
```

Comment:
This check should be earlier (at the condition `if config.vars.cat.characters`). The report renderer is already able to deal with empty unicode sets. 

If unicode is available, but the library is not installed, then a warning should be given.

Reply:
@sbrugman Sorry, what do you mean with "if unicode is available"? 

Do you mean that if `config.vars.cat.characters` is enabled and tangled isn't, we should raise a warning and not do any of the character checks? Since `unicodedata` (built-in in Python) provides `category`, we can always provide category data if nothing else.

Reply:
@akx I do agree with you. I don't think we should raise a warning in this situation given that we have built-in Python `unicodedata`.

Reply:
For future reference: the builtin `unicodedata`'s category returns a category code. The `category_long` in `tangled-up-in-unicode` looks up the more descriptive longer version useful for users. Might be worth adding a link/hint when some unicode properties are uncomputed.